grakn-engine/src/main/java/ai/grakn/engine/tasks/manager/multiqueue/MultiQueueTaskManager.java

```diff
@@ -74,6 +77,7 @@ public MultiQueueTaskManager(EngineID engineId) {
 
         this.producer = ConfigHelper.kafkaProducer();
         EngineCacheProvider.init(EngineCacheDistributed.init(zookeeper));
+        LockProvider.init(new ZookeeperLock(zookeeper, LOCK));
```

Comment:
Yep that needs to change then. I would prefer for that change to happen in this PR but since we in a bit of a hurry we could put this in and get the updating counts task using it. This will make PP unsafe to run on the cluster but we could disable PP for the quick load. It means duplicates will be seen though.